bundle/config/loader/process_root_includes.go

```diff
@@ -39,6 +51,23 @@ func (m *processRootIncludes) Apply(ctx context.Context, b *bundle.Bundle) diag.
 	var files []string
 	var diags diag.Diagnostics
 
+	// We error on glob characters in the bundle root path since they are
+	// parsed by [filepath.Glob] as being glob patterns and thus can cause
+	// unexpected behavior.
+	//
+	// The standard library does not support globbing from a relative path,
```

Comment:
What does this mean? You can apply glob on "*.whl" for example, which is relative path.

Reply:
Thus I recommend erroring here to keep complexity lower.